modules/kms-keyring-browser/src/kms_mrk_discovery_keyring_browser.ts

```diff
@@ -16,7 +16,7 @@ import {
   importForWebCryptoDecryptionMaterial,
   KeyringWebCrypto,
   Newable,
-} from '@aws-crypto/material-management-browser'
+} from '@symon-ai/aws-crypto-material-management-browser'
```

Comment:
there used to be a bunch of changes in these files referencing kmsclient for sdk-v3, but now this code uses a interface, so it no longer references v2 or v3 directly.